deployments/deployment.yaml

```diff
@@ -213,6 +213,7 @@ objects:
       database:
         name: content-sources
         version: 13
+      inMemoryDb: true
```

Comment:
Does this give you a elasticache?

Reply:
in ephemeral its a redis instance i believe, but in stage/prod it'll be elasticache.  I believe there is an app-interface change i'll need to make after merge to make it actually available in stage